.prerequisites.py

```diff
@@ -138,9 +156,16 @@ def run_checks(check_list, exclusions=[]):
 
 def main():
     checks = both_checks
-    if osx():
-        print('OSX detected')
+    platform = sys.platform
+    if platform.startswith('linux'):
+        print('Linux detected')
+        checks = linux_checks + checks
+    elif platform == 'darwin': 
+        print('Mac OS detected')
         checks = mac_checks + checks
+    else:
+        print('Unsupported platform %s' % platform)
+        exit(1)
```

Comment:
Remove this `exit` please. Feel free to replace it with a warning about 'unexpected behaviour' or similar.